### PR TITLE
Add release:list command

### PR DESCRIPTION
**Pull request by fson**

### CODE REVIEW

File: packages/eas-cli/src/commands/release/list.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    if (flags.json) {
+      log(JSON.stringify(releases, null, 2));
+    } else {
+      const table = new CliTable({ head: ['release', 'update', 'updated-at', 'user'] });
+      table.push(
+        ...releases.map((release: UpdateRelease) => [
+          release.releaseName,
+          release.updates[0]?.id,
+          release.updates[0]?.updatedAt,
+          release.updates[0]?.actor?.firstName,
```
Comment (jkhales) [inline packages/eas-cli/src/commands/release/list.ts:77]: If there is room, adding the `updateMessage` would help readability a lot.

File: packages/eas-cli/src/commands/release/list.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    const fullName = await getProjectFullNameAsync(projectDir);
+    const releases = await listReleasesAsync({ fullName });
+    if (flags.json) {
+      log(JSON.stringify(releases, null, 2));
+    } else {
+      const table = new CliTable({ head: ['release', 'update', 'updated-at', 'user'] });
+      table.push(
+        ...releases.map((release: UpdateRelease) => [
+          release.releaseName,
+          release.updates[0]?.id,
```
Comment (jkhales) [inline packages/eas-cli/src/commands/release/list.ts:75]:
```suggestion
          release.updates[0]?.updateGroup,
```
There is one `update.id` for every platform a user publishes to. 
There is one `updateGroup` for every publish.

File: packages/eas-cli/src/commands/release/list.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  const data = await withErrorHandlingAsync(
+    graphqlClient
+      .query<RootQuery>(
+        gql`
+          query ReleasesByAppQuery($fullName: String!) {
+            app {
+              byFullName(fullName: $fullName) {
+                id
+                fullName
+                updateReleases(offset: 0, limit: 10000) {
```
Comment (jkhales) [inline packages/eas-cli/src/commands/release/list.ts:20]: What is the plan for if there are more than 10,000? Should we log a warning if the number of releases is >= 10,000? `Returning the max number of releases, 10,000`

File: packages/eas-cli/src/commands/release/list.ts [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    const projectDir = await findProjectRootAsync(process.cwd());
+    if (!projectDir) {
+      throw new Error('Please run this command inside a project directory.');
+    }
+    const fullName = await getProjectFullNameAsync(projectDir);
+    const releases = await listReleasesAsync({ fullName });
+    if (flags.json) {
+      log(JSON.stringify(releases, null, 2));
+    } else {
+      const table = new CliTable({ head: ['release', 'update', 'updated-at', 'user'] });
```
Comment (jkhales) [inline packages/eas-cli/src/commands/release/list.ts:71]:
```suggestion
      const table = new CliTable({ head: ['release', 'current update', 'updated-at', 'user'] });
```

**Comment by fson** [comment]
Screen Shot 2020-12-21 at 18 22 30

Added more concise formatting of updates.

File: packages/eas-cli/src/commands/release/list.ts [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+import gql from 'graphql-tag';
+import { format } from 'timeago.js';
+
+import { graphqlClient } from '../../graphql/client';
+import { RootQuery, Update, UpdateRelease } from '../../graphql/generated';
+import log from '../../log';
+import { findProjectRootAsync, getProjectFullNameAsync } from '../../project/projectUtils';
+import { getActorDisplayName } from '../../user/actions';
+
+const RELEASES_LIMIT = 10000;
```
Comment (jkhales) [inline packages/eas-cli/src/commands/release/list.ts:14]:
```suggestion
const RELEASES_LIMIT = 10_000;
```

**Review by jkhales — APPROVED** [review]
Looks great! 🚀

**Pull request merged**